components/image-input/image.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="../../assets/light-favicon.png"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="image.css" />
+
+    <link rel="stylesheet" href="../../docs.css" />
+    <title>Light UI</title>
+  </head>
+  <body>
+    <!-- ui nav bar -->
+    <nav class="ui-nav">
+      <div class="right-nav">
+        <span class="main-heading">Light UI</span>
+        <span class="sub-heading">v 1.0</span>
+      </div>
+      <ul class="left-nav">
+        <div class="nav-item"><a href="#">Example</a></div>
+        <div class="nav-item"><a href="#">Github</a></div>
+      </ul>
+    </nav>
+    <!-- ui nav bar -->
+    <div class="main-box">
+      <!-- docs side bar -->
+      <div class="side-bar">
+        <div class="sidebar-title">Components</div>
+        <div class="sidebar-item">
+          <a class="link-text" href="../typography/typo.html">Typography</a>
+        </div>
+        <div class="sidebar-item">
+          <a class="link-text" href="avtar.html">Avatar</a>
+        </div>
+        <div class="sidebar-item"><a class="link-text" href="#">Badge</a></div>
+        <div class="sidebar-item">
+          <a class="link-text" href="../alert-btn/alert.html">Alert</a>
+        </div>
+        <div class="sidebar-item">
+          <a class="link-text" href="../alert-btn/button.html">Button</a>
+        </div>
+        <div class="sidebar-item">
+          <a class="link-text" href="#">Image</a>
+        </div>
+        <div class="sidebar-item">
+          <a class="link-text" href="input.html">Input</a>
+        </div>
+        <div class="sidebar-item">
+          <a class="link-text" href="../card/card.html">Card</a>
+        </div>
+      </div>
+      <!-- docs side bar -->
+      <!-- docs main contant -->
+      <div class="doc-contant">
+        <div class="docs-info">
+          <div class="docs-heading">Images</div>
+          <div class="docs-sub-heading">
+            Images can be responsive to fit the parent's width, and also can be
+            customised to be round shaped
+          </div>
+        </div>
+        <div class="docs-info">
+          <div class="docs-heading doc-m-heading">
+            Example of Responsive Image
+          </div>
+          <div class="sub-heading">
+            You can add class <span class="highlight">responsive-img</span> to
+            make your image fit the width of container. It's height will get
+            adjusted by keeping the aspect ratio same. If you want to change the
+            aspect ratio, you will have to crop the image.
+          </div>
+          <div class="demo-component-typo">
+            <img
+              class="responsive-img"
+              src="/assets/res-demo-img.jpeg"
```

Comment:
This image is taking a lot of time for me to load. It's more than 1 MB in size. So maybe try a lighter version

Reply:
it is fixed now